templates/config.edn

```diff
@@ -1,137 +1,160 @@
 {:meta/version 1
 
- ;; Currently, we support either "Markdown" or "Org".
- ;; This can overwrite your global preference so that
- ;; maybe your personal preferred format is Org but you'd
- ;; need to use Markdown for some projects.
- ;; :preferred-format ""
-
- ;; Preferred workflow style.
- ;; Value is either ":now" for NOW/LATER style,
- ;; or ":todo" for TODO/DOING style.
+ ;; Set the preferred format.
+ ;; Available options:
+ ;; - "Markdown" (default)
+ ;; - "Org"
+ :preferred-format "Markdown"
+
+ ;; Set the preferred workflow style.
+ ;; Available options:
+ ;; - ":now" for NOW/LATER style (default)
+ ;; - ":todo" for TODO/DOING style
  :preferred-workflow :now
 
- ;; The app will ignore those directories or files.
- ;; E.g. :hidden ["/archived" "/test.md" "../assets/archived"]
+ ;; Exclude directories/files.
+ ;; Example usage: 
+ ;; :hidden ["/archived" "/test.md" "../assets/archived"]
  :hidden []
 
- ;; When creating the new journal page, the app will use your template if there is one.
- ;; You only need to input your template name here.
- :default-templates
- {:journals ""}
+ ;; Define the default journal page template.
+ ;; Enter the template name between the quotes.
+ :default-templates {:journals ""}
 
- ;; Set a custom date format for journal page title
- ;; Example:
- ;; :journal/page-title-format "EEE, do MMM yyyy"
+ ;; Set a custom date format for the journal page title.
+ ;; Default value: "MMM do, yyyy"
+ :journal/page-title-format "MMM do, yyyy"
+
+ ;; Specify the journal filename format using a valid date format string.
+ ;; !Warning:
+ ;;   This configuration is not retroactive and affects only new journals.
+ ;;   To show old journal files in the app, manually rename them to the new pattern.
+ ;; Default value: "yyyy-MM-dd"
+ :journal/file-name-format "yyyy-MM-dd"
```

Comment:
Hey @Bad3r , the default format here should be 
`yyyy_MM_dd`
https://github.com/logseq/logseq/blob/c76da13570c7f9d2f8374416e428267cd16d46ba/src/main/frontend/date.cljs#L186

Uncomment is some kinda dangerous operation. All values requires double check.